tests/pytests/unit/client/test_init.py

```diff
@@ -0,0 +1,271 @@
+import pytest
+
+import salt.client
+from salt.exceptions import SaltInvocationError
+
+
+@pytest.fixture
+def local_client():
+    return salt.client.get_local_client()
+
+
+def test_get_local_client(local_client):
+    assert isinstance(local_client, salt.client.LocalClient)
+
+
+def test_get_local_client_mopts(master_opts):
+    master_opts["rest_cherrypy"] = {"port": 8000}
+    local_client = salt.client.get_local_client(mopts=master_opts)
+    assert isinstance(local_client, salt.client.LocalClient)
```

Comment:
We changed the arguments here but are only checking the instance. Is there a way to assert that passing in different opts changed the behavior of the code?

Reply:
Added an additional assert that the opts match